core/src/main/java/org/apache/gravitino/catalog/PropertiesMetadataHelpers.java

```diff
@@ -54,25 +54,31 @@ public static void validatePropertyForCreate(
             .collect(Collectors.toList());
     Preconditions.checkArgument(
         reservedProperties.isEmpty(),
-        "Properties are reserved and cannot be set: %s",
+        "Properties or properties with a fixed prefix are reserved and cannot be set: %s",
         reservedProperties);
 
     List<String> absentProperties =
-        propertiesMetadata.propertyEntries().keySet().stream()
-            .filter(propertiesMetadata::isRequiredProperty)
-            .filter(k -> !properties.containsKey(k))
+        propertiesMetadata.propertyEntries().values().stream()
+            .filter(PropertyEntry::isRequired)
+            .filter(
+                e ->
+                    (!e.isPrefix() && !properties.containsKey(e.getName()))
+                        || (e.isPrefix()
+                            && properties.keySet().stream()
+                                .noneMatch(k -> k.startsWith(e.getName()))))
+            .map(PropertyEntry::getName)
             .collect(Collectors.toList());
     Preconditions.checkArgument(
         absentProperties.isEmpty(),
-        "Properties are required and must be set: %s",
+        "Properties or properties with a fixed prefix are required and must be set: %s",
```

Comment:
I can't get `Properties or properties with a fixed prefix are required`. According to your code, it should be `Required properties should be set or required prefix properties should be set with prefixes in the key`.

Reply:
To be more accurate, `prefix properties` should be `a property with a fixed prefix`, then you can understand the above description well